pom.xml

```diff
@@ -106,5 +106,11 @@
             <version>3.9.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.9.4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
whats this for?

Reply:
I'm not too sure, I know jackson is used for parsing JSON, and I do parse JSON in my code but on uncommenting it my tests still run, once I'm done implementing the requested changes I'll check if Travis is affected by removing it